lib/src/transaction/msg/ibc/applications/MsgTransfer.spec.ts

```diff
@@ -158,8 +142,47 @@ describe('Testing MsgTransfer', function () {
             timeoutHeight,
             timeoutTimestamp: Long.fromString('1620640362229420996'),
         });
+        expect(isAminoMsgTransfer(MsgTransfer.toRawAminoMsg()).valueOf()).to.be.true;
+        expect(MsgTransfer.toRawAminoMsg()).to.deep.eq({
+            type: 'cosmos-sdk/MsgTransfer',
+            value: {
+                receiver: 'cosmos1vw4ucaeagtduv5ep4sa95e3aqzqpsk5meda08c',
+                sender: 'tcro15sfupd26sp6qf37ll5q6xuf330k7df9tnvrqht',
+                source_channel: 'channel-33',
+                source_port: 'transfer',
+                timeout_height: {
+                    revision_height: '1708515',
+                    revision_number: '0',
+                },
+                timeout_timestamp: '1620640362229420996',
+                token: {
+                    amount: '1234',
+                    denom: 'basetcro',
+                },
+            },
+        });
+
+        // Case when timeoutHeight and token is undefined
+        const msgTransferIBC_UNdefined = new cro.ibc.MsgTransfer({
+            sourcePort: 'transfer',
+            sourceChannel: 'channel-33',
+            sender: 'tcro15sfupd26sp6qf37ll5q6xuf330k7df9tnvrqht',
+            receiver: 'cosmos1vw4ucaeagtduv5ep4sa95e3aqzqpsk5meda08c',
+            timeoutTimestamp: Long.fromString('1620640362229420996'),
```

Comment:
```suggestion
            timeoutTimestampInNanoSeconds: Long.fromString('1620640362229420996'),
```

This is the confusing part I wanted to highlight, we would need to specify it expected in nanoseconds, that's because traditionally timestamp is usually in milli seconds.

Reply:
Yes, I have renamed the user end to signify that it's expected in nano Seconds.

Or do you mean something else?